test/e2e/velero_utils.go

```diff
@@ -327,7 +363,14 @@ func veleroCreateBackupLocation(ctx context.Context,
 // veleroAddPluginsForProvider determines which plugins need to be installed for a provider and
 // installs them in the current Velero installation, skipping over those that are already installed.
 func veleroAddPluginsForProvider(ctx context.Context, veleroCLI string, veleroNamespace string, provider string) error {
-	for _, plugin := range getProviderPlugins(provider) {
+	var plugins []string
+	var err error
+	version := "latest"
```

Comment:
Get the version from parameter or cli rather than hard code as `latest` here

Reply:
We may need to introduce a new parameter for the plugins used by the additional BSL